docs/store.js

```diff
@@ -210,21 +207,21 @@ var firebaseCalendarStorage = function(params) {
         
         self.startWork();
         return self.database.ref('users/' + userId + '/allCalendarIds/' + uniqueId).remove()
-        .then(function() {
-            self.database.ref('calendars/' + uniqueId).remove()
             .then(function() {
-                self.endWork();
+                self.database.ref('calendars/' + uniqueId).remove()
+                .then(function() {
+                    self.endWork();
+                })
+                .catch(function(err) {
+                    console.error('Unable to remove calendar: ' + err);
+                    self.endWork();
+                })
             })
             .catch(function(err) {
-                console.error('Unable to remove calendar: ' + err);
+                console.error('Unable to remove calendar from user\'s allCalendarIds: ' + err);
```

Comment:
It's usually easier to read if you make use of the other type of quotes so that you don't have to escape.